src/test/regress/multi_schedule

```diff
@@ -176,5 +176,5 @@ test: multi_expire_table_cache
 
 # ----------
 # multi_colocation_utils tests utility functions written for co-location feature & internal API
+# multi_colocated_shard_transfer tests master_copy_shard_placement with colocated tables.
 # ----------
```

Comment:
Why we;ve removed `multi_colocation_utils` and not added `multi_colocated_shard_transfer` to multi_schedule? Do I miss anything?